packages/viewer/src/scenes/scene.ts

```diff
@@ -3,6 +3,75 @@ import { Frame } from '../types';
 import { Camera } from './camera';
 import { Dimensions } from '@vertexvis/geometry';
 import { Raycaster } from './raycaster';
+import { ColorMaterial, fromHex } from './colorMaterial';
+import { SceneItemOperations, SceneOperationBuilder } from './operations';
+import {
+  SceneItemQuery,
+  ItemSelector,
+  Selector,
+  ItemSelectorBuilder,
+} from './selectors';
+import { CommandRegistry } from '../commands/commandRegistry';
+import { UUID } from '@vertexvis/utils';
+
+export class SceneItemOperationsExecutor
+  implements SceneItemOperations<SceneItemOperationsExecutor> {
+  protected builder = new SceneOperationBuilder();
+  public constructor(
+    private sceneViewId: UUID.UUID,
+    private commands: CommandRegistry,
+    private query: ItemSelectorBuilder
+  ) {}
+
+  public materialOverrideFromHex(color: string): SceneItemOperationsExecutor {
+    return this.materialOverride(fromHex(color));
+  }
```

Comment:
Unless there's a strong reason to, my preference would be to not have these helpers. These helpers could get pretty numerous, and would be better to organize them on the material type, e.g. `ColorMaterial`.